src/Cron_Event_Command.php

```diff
@@ -441,26 +444,28 @@ private static function interval( $since ) {
 
 		// step one: the first chunk
 		for ( $i = 0, $j = count( $chunks ); $i < $j; $i++ ) {
-			$seconds = $chunks[$i][0];
-			$name = $chunks[$i][1];
+			$seconds = $chunks[ $i ][0];
+			$name    = $chunks[ $i ][1];
 
 			// finding the biggest chunk (if the chunk fits, break)
-			if ( ( $count = floor( $since / $seconds ) ) != 0 ){
+			$count = floor( $since / $seconds );
+			if ( floatval( 0 ) !== $count ) {
 				break;
 			}
 		}
 
 		// set output var
-		$output = sprintf( \_n( $name[0], $name[1], $count ), $count );
+		$output = sprintf( '%d %s', $count, Utils\pluralize( $name, absint( $count ) ) );
 
 		// step two: the second chunk
 		if ( $i + 1 < $j ) {
-			$seconds2 = $chunks[$i + 1][0];
-			$name2    = $chunks[$i + 1][1];
+			$seconds2 = $chunks[ $i + 1 ][0];
+			$name2    = $chunks[ $i + 1 ][1];
 
-			if ( ( $count2 = floor( ( $since - ( $seconds * $count ) ) / $seconds2 ) ) != 0 ) {
+			$count2 = floor( ( $since - ( $seconds * $count ) ) / $seconds2 );
+			if ( floatval( 0 ) !== $count2 ) {
 				// add to output var
-				$output .= ' ' . sprintf( \_n( $name2[0], $name2[1], $count2 ), $count2 );
+				$output .= ' ' . sprintf( '%d %s', $count2, Utils\pluralize( $name2, absint( $count2 ) ) );
```

Comment:
If `pluralize()` comes from `WP_CLI\Utils`, the function call does not need the `Utils\` part as you've added the `use` statement.